docs/data/toolpad/core/components/account/AccountSlotsWallet.js

```diff
@@ -0,0 +1,126 @@
+import * as React from 'react';
+import {
+  Avatar,
+  Button,
+  Divider,
+  Typography,
+  Stack,
+  IconButton,
+} from '@mui/material';
+import WalletIcon from '@mui/icons-material/AccountBalance';
+import SendIcon from '@mui/icons-material/Send';
+import ShoppingCart from '@mui/icons-material/ShoppingCart';
+import CopyIcon from '@mui/icons-material/ContentCopy';
+
+import {
+  Account,
+  AccountPreview,
+  AuthenticationContext,
+  SessionContext,
+  SignOutButton,
+} from '@toolpad/core';
+import { AccountPopoverFooter } from '../../../../../../packages/toolpad-core/src/Account/AccountPopoverFooter';
+
+const demoSession = {
+  user: {
+    name: 'Bharat Kashyap',
+    email: 'bharatkashyap@outlook.com',
+    image: 'https://avatars.githubusercontent.com/u/19550456',
+  },
+};
+
+const mockData = {
+  address: '0xb794f5ea0ba39494ce839613fffba74279579268',
+  balance: '1,234.56 ETH',
+  usdBalance: '$2,345,678.90 USD',
+};
+
+function CryptoWalletInfo() {
+  return (
+    <Stack direction="column">
+      <AccountPreview variant="expanded" />
+      <Divider />
+      <Stack spacing={2} sx={{ width: 300, p: 2 }}>
+        <Stack direction="row" spacing={2} alignItems="center">
+          <Avatar sx={{ bgcolor: 'primary.main' }}>
+            <WalletIcon />
+          </Avatar>
+          <Stack sx={{ flexGrow: 1, minWidth: 0 }}>
+            <Typography
+              variant="subtitle1"
+              fontWeight="bold"
+              sx={{
+                display: 'flex',
+                alignItems: 'center',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                whiteSpace: 'nowrap',
+              }}
+            >
+              <span style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>
+                {mockData.address}
+              </span>
+              <IconButton size="small" sx={{ ml: 1 }}>
+                <CopyIcon fontSize="small" />
+              </IconButton>
+            </Typography>
+            <Typography variant="body2" color="text.secondary">
+              Main Account
+            </Typography>
+          </Stack>
+        </Stack>
+        <Divider />
+        <Stack>
+          <Typography variant="h6" fontWeight="bold">
+            {mockData.balance}
+          </Typography>
+          <Typography variant="body2" color="text.secondary">
+            {mockData.usdBalance}
+          </Typography>
+        </Stack>
+      </Stack>
+      <Divider />
+      <AccountPopoverFooter sx={{ gap: 2, px: 2, justifyContent: 'center' }}>
```

Comment:
I thought that we were going to have a separate slot for customizing the header or footer, for example, instead of just one slot for all the popover content, and it felt more intuitive for me to use and more consistent with how MUI generally allows this kind of customization, as opposed to an `AccountPopoverFooter` component that takes children.

Thse components by themselves seem like really simple generic wrappers too.

This might also mean that maybe `signOutButton` shouldn't be a slot, just a component that can optionally be imported to be used/composed in the footer slot.

Also in this example in particular the preview isn't helpful enough, I would need to expand and find `AccountPopoverFooter` to see how it's being used. Maybe the preview could be more helpful somehow?

Reply:
Oh, I just saw that the `PageContainer` has a similar approach to this so anyway it was just my opinion.
Not strongly against doing it this way in the end.

Reply:
Yep, we were going by the `PageContainerToolbar` way of doing things since it is also the new way of composition/customization that X is adopting; lesser slots and more wrappable/composable components

Reply:
Got it, sounds good then!